SELF_HOSTED_INSTRUCTIONS.md

```diff
@@ -0,0 +1,48 @@
+## First one time set up
+Make sure you have running uwazi [dependencies](https://github.com/huridocs/uwazi#dependencies).
```

Comment:
Revert the verb with the adjective: `Ensure you have Uwazi dependencies running and correctly configured`.  Maybe the link is "Uwazi dependencies"? Be sure to use capital U in Uwazi.